_posts/2019-11-13-AWS-Dev-Day-2019.md

```diff
@@ -0,0 +1,76 @@
+
+All of this gives you the best toolkit to NOT create snowflake servers.
+
+The CDK also has a CLI which can be installed by running `npm install -g aws-cdk`. 
+
+### Pipeline
+Amazon's pipeline service is called [AWS CodePipeline](https://aws.amazon.com/codepipeline/){:target="_blank" rel="noopener noreferrer"}. This service allows you to create a pipeline which executes all the necessary steps to automate your releases and to provide infrastructure updates. This allows you to quickly provide new releases and deliver new features to your end users. You can integrate this service with other external services (ex. Github, where your repository is hosted). 
+
+To effectively build your application, you can make use of [AWS CodeBuild](https://aws.amazon.com/codebuild/){:target="_blank" rel="noopener noreferrer"}. This is their continuous integration service that packages your application by building it and running the tests. Pricing is very effective, as you will only pay by the minute that you trigger a build. CodeBuild automatically scales and can process multiple builds at the same time, so you won't have to deal with build queues. 
+
+### Deploy
+[AWS CodeDeploy](https://aws.amazon.com/codedeploy/){:target="_blank" rel="noopener noreferrer"} is a deployment service which basically allows you to deploy (almost) anywhere, including (but not limited to) on-premise servers. This service makes it easier to provide updates to your end users and to avoid downtime during the rollout of a new update. 
+
+One of the most impressive features is the Blue/Green deployment. This splits the traffic going to your application into traffic to your ortiginal environment and an other environment. The best use case for this would be during a new production release. You have a new release that you want to deploy? Simply let 10% of your traffic go to that release to make sure everything works right. After monitoring the release, you can choose to either revert or to fully allow traffic to go to the newest release.
+
+### Conclusion
+Amazon provides a lot of tools that help a project to manage their builds and deploys, making sure that everything works according to plan. We were definitely a fan of their new Cloud Development Kit and their Blue/Green deployment is able to stabilize and fully control the rollout of a new deployment.
+
+# Integrate your front end apps with serverless backend in the cloud by Sebastian Stormacq
+Serverless is a big trend nowadays and rightfully so, as it does a lot of stuff for you. The biggest pros of serverless development are that you no longer need to provision or manage your own server, it automatically scales with the amount of usage, it has built in fault tolerance and fallback methods and last but not least, you only pay for the value. This means that if you are not using your lambda, you are not paying for it!
```

Comment:
Serverless is  a big trend nowadays and rightfully so as it takes away a lot of operational work from you.